doc/users/resources/index.rst

```diff
@@ -11,6 +11,10 @@ External resources
 Books, chapters and articles
 ============================
 
+* `Scientific Visualization: Python + Matplotlib (2021)
+   <https://hal.inria.fr/hal-03427242/>`
```

Comment:
Could the missing underscore at the end be why the docs aren't building?

Reply:
Oh s**t, sorry for that. I'll correct it.

Reply:
Lesson learned. Even a simple PR needs to be tested.

Reply:
there's a white space failure 
![image](https://user-images.githubusercontent.com/1300499/170888866-9e2d97db-4880-42a7-8cea-1f67e5e4a2fc.png)
and it's still not building correctly :/